Completed/app/scripts/app.js

```diff
@@ -35,38 +35,19 @@ function onAppActivate() {
  * Function to Fetch location of International Space Station from the REST API
  * 
  */
-function fetchData() {
+async function fetchData() {
 
   // API endpoint to fetch the current location of the International Space Station
-  const API_BASE_URL = "https://api.wheretheiss.at/v1/satellites/25544/positions"
+  // const API_BASE_URL = "https://api.wheretheiss.at/v1/satellites/25544/positions"
```

Comment:
This line can be deleted if not required anymore. If just required for a reference, add it to the comments section.

Reply:
Removed the base URL.